packages/sd-jwt-vc/src/sd-jwt-vc-instance.ts

```diff
@@ -306,18 +307,18 @@ export class SDJwtVcInstance extends SDJwtInstance<SdJwtVcPayload> {
           StatusListJWTPayload
         >(statusListJWT);
         // check if the status list has a valid signature. The presence of the verifier is checked in the parent class.
-        await slJWT.verify(
-          this.userConfig.statusVerifier ??
-            (this.userConfig.verifier as Verifier),
-          options,
-        );
-
-        const currentDate =
-          options?.currentDate ?? Math.floor(Date.now() / 1000);
-        //check if the status list is expired
-        if (slJWT.payload?.exp && (slJWT.payload.exp as number) < currentDate) {
-          throw new SDJWTException('Status list is expired');
-        }
+        await slJWT
+          .verify(
+            this.userConfig.statusVerifier ??
+              (this.userConfig.verifier as Verifier),
+            options,
+          )
+          .catch((err: SLException) => {
```

Comment:
I think casting this as an `SLException` is unsafe, since the method could throw other errors as well (since a custom `statusVerifier` can be passed).